app/src/main/java/com/fish/monsters/features/safetyInformation/SafetyInformationScreen.kt

```diff
@@ -0,0 +1,140 @@
+package com.fish.monsters.features.safetyInformation
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Checkbox
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.fish.monsters.R
+import com.fish.monsters.common.views.PreviewContainer
+import com.fish.monsters.common.views.SafetyInformationCard
+import com.fish.monsters.common.views.buttons.FishButton
+import com.fish.monsters.common.views.screenContent.ScreenBox
+import com.fish.monsters.core.navigation.Navigator
+import com.fish.monsters.core.navigation.Screen
+import com.fish.monsters.core.theme.TextColorLight
+import com.fish.monsters.features.preferences.PreferencesViewModel
+
+@Composable
+fun SafetyInformationScreen(
+    navigator: Navigator?,
+    preferencesViewModel: PreferencesViewModel?
```

Comment:
to nie powinno być nullable, zamiast tego musisz to wstrzyknąć z koina (po dodaniu `PreferencesViewModel` do DI) w ten sposób:
```
navigator: Navigator = koinInject(),
preferencesViewModel: PreferencesViewModel = koinInject()
```